### PR TITLE
Swallow IOError, but not ImportError

### PR DESCRIPTION
**Pull request by jessesnyder**
## Description
See https://github.com/Dallinger/Dallinger/issues/602

## How Has This Been Tested?
No new automated tests. Tested manually.

### CODE REVIEW

**Comment by jessesnyder** [comment]
@MatthewWilkes Not sure if I may have missed some implication here, so assigning you as a reviewer.

**Comment by jessesnyder** [comment]
@MatthewWilkes I went ahead with this merge, but would love your blessing (or condemnation) when you're "back in the saddle", as we Yanks say.

**Comment by alecpm** [comment]
@jessesnyder this looks fine to me FWIW.

**Comment by MatthewWilkes** [comment]
@jessesnyder I've done some playing with this, and I think it is okay to not handle `ImportError`, as it's clearer that the problem is that there are missing dependencies that are preventing the command from working, rather than simply not loading additional config variables and waiting for a crash later in the process.

That said, I do think we're going to have problems as more and more experiments are created, as we're really blurring the line between experiment dependencies and Dallinger dependencies in this situation. I don't think we should ever be parsing the `experiment.py` file not in the context of a virtual environment that was created by its `requirements.txt`, otherwise we're encouraging people to install particular experiment dependencies into the interpreter that has Dallinger in it (which may even be the system environment).

TL;DR: I'm +1 on this change, but think we need to do some work on dependency management in general.
